src/logic/operate.js

```diff
@@ -0,0 +1,34 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numOne = Big(numberOne);
+  const numTwo = Big(numberTwo);
+
+  let total;
+
+  if (operation === '+') {
+    total = numOne.plus(numTwo).toString();
+  } else if (operation === '-') {
+    total = numOne.minus(numTwo).toString();
+  } else if (operation === '×') {
+    total = numOne.times(numTwo).toString();
+  } else if (operation === '÷') {
+    if (numTwo.eq(0)) {
+      total = 'Division by zero is undefined';
```

Comment:
I think assigning this type of value to `total` may result to another error with further operations since instantiating `Big` with this type of string will result to an `Uncaught Error [BigError]: NaN`. Below is an example of instantiating Big with the above string.

```js
// node
const Big = require("big-js");
Big('Division by zero is undefined')
Uncaught Error [BigError]: NaN
    at throwErr (/workspace/react-calculator/node_modules/big-js/big.js:314:19)
    at parse (/workspace/react-calculator/node_modules/big-js/big.js:184:13)
    at new Big (/workspace/react-calculator/node_modules/big-js/big.js:90:17)
    at Big (/workspace/react-calculator/node_modules/big-js/big.js:81:54)
```

Reply:
Removed the string to avoid errors with further operations.